src/ducks/markets.js

```diff
@@ -88,6 +88,23 @@ export const getFilteredMarkets = createSelector(
 			}))
 );
 
+export const getAllMarkets = createSelector(
+	getMarketsList,
+	getRatesExchangeRates,
+	(marketsList, exchangeRates) =>
+		marketsList.map(market => ({
+			...market,
+			lastPrice:
+				exchangeRates == null
+					? null
+					: getExchangeRatesForCurrencies(
+							exchangeRates,
+							market.baseCurrencyKey,
+							market.quoteCurrencyKey
+					  ) || 0,
+		}))
+);
+
```

Comment:
Minor - this seems like a duplicate mapping functionality, both `getFilteredMarkets` and `getAllMarkets` should use some function to map the exchange rates

Reply:
Yeah good call.